tasks/internal/request-and-upload-signature/tests/test-request-and-upload-signature-failed-messages.yaml

```diff
@@ -0,0 +1,113 @@
+spec:
+  description: |
+    Run the request-and-upload-signature task with use case when signer return error in messages. 
+  workspaces:
+    - name: tests-workspace
+  tasks:
+    - name: setup
+      workspaces:
+        - name: data
+          workspace: tests-workspace
+      taskSpec:
+        workspaces:
+          - name: data
+        steps:
+          - name: setup-values
+            image: quay.io/konflux-ci/release-service-utils:e633d51cd41d73e4b3310face21bb980af7a662f
+            script: |
+              #!/usr/bin/env bash
+              set -eux
+              cat << EOF | jq -Mc > "$(workspaces.data.path)/mocked_signing_response"
+              {"signer_result": {"status": "ok", "error_message": ""},
+                     "operation_results": [
+                       [
+                         {"i": 1,
+                          "msg": {
+                            "errors": ["failed"],
+                            "manifest_digest": "sha256:123456",
+                            "pub_task_id": "123456",
+                            "repo": "test-repo",
+                            "request_id": "1",
+                            "request_received_time": "3000-01-01T01:01:01.000000",
+                            "requested_by": "test-user",
+                            "sig_key_id": "test-signing-key",
+                            "sig_keyname": "test-signing-key",
+                            "signature_type": "container_signature",
+                            "signed_claim": "signed-data",
+                            "signing_server_requested": "3000-01-01T01:01:01.000000",
+                            "signing_server_responded": "3000-01-01T01:01:02.000000",
+                            "signing_status": "success"
+                          },
+                         "msg_id": "msg-id-1",
+                         "timestamp": 1738685708,
+                         "topic": "/topic/VirtualTopic.eng.test.dist.sign.123456",
+                         "username": "1001030000"
+                        }, 
+                        {"amq6100_originalDestination": "topic://VirtualTopic.eng.test.dist.sign.123456",
+                         "content-type": "text/plain",
+                         "amq6100_destination": "queue://Consumer.test-user.123456.VirtualTopic.eng.test.dist.sign.123456"
+                        }
+                       ]
+                     ],
+                     "operation": {"digests": ["sha256:123456"],
+                                   "references": ["registry.redhat.io/myproduct/myrepo:abc"],
+                                   "signing_key": "test-signing-key", "task_id": "123456"},
+                     "signing_key": "test-signing-key"
+              }
+              EOF
+              touch "$(workspaces.data.path)/mock_pubtools-pyxis-upload-signatures.txt"
+              touch "$(workspaces.data.path)/signing_response.json"
+              chmod 0777 -R "$(workspaces.data.path)"
```

Comment:
I'm curious why this is needed. Was there some permission issue otherwise?

Reply:
yeah, that was fixing just a problem with my local setup. I'll remove it.